src/tutorial/cli-args.md

```diff
@@ -116,7 +116,7 @@ and it’ll generate the code that parses the arguments into the fields.
 [`structopt`]: https://docs.rs/structopt
 
 Let's first import `structopt` by adding
-`structopt = "0.3.13"` to the `[dependencies]` section
+`structopt = "0.3"` to the `[dependencies]` section
```

Comment:
> Allows readers to benefits from latest bug fixes (10 versions) while avoiding breaking-change updates that would make the book obsolete.

I think there might be a misunderstanding here on Cargo version requirements

``structopt = "0.3.13"` is shorthand for `structopt = "^0.3.13"` which is shorthand for `structopt = ">=0.3.13,<0.4"`

See https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html

Effectively this means "use a semver compatible version that is at least 0.3.13"

Reply:
Oops my bad! `0.3.13` seemed arbitrary though.